tests/NPSP.py

```diff
@@ -706,5 +706,12 @@ def verify_opportunities(self, len_value):
                 self.selenium.page_should_contain_element(loctor_contains)
                 
     
-            
+    def page_scroll_to_locator(self, path, *args, **kwargs):
+        loc=self.get_npsp_locator(path, *args, **kwargs)
+        self.selenium.execute_javascript("window.document.evaluate('{}', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.scrollIntoView(true)".format(loc))
```

Comment:
In cumulusci 2.1.1 we upgraded robotframework-seleniumlibrary to a version that includes a "Scroll Element Into View" keyword. So instead of doing this with javascript, we should do `self.selenium.scroll_element_into_view(loc)`